ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,225 @@
+
+
+class CreateVerb(VerbExtension):
+    """Create a new ROS2 package."""
+
+    def add_arguments(self, parser, cli_name):
+        parser.add_argument(
+            'package_name',
+            help='The package name')
+        parser.add_argument(
+            '--destination-directory',
+            default=os.curdir,
+            help='Directory where to create the package')
+        parser.add_argument(
+            '--build-tool',
+            default='ament_cmake',
+            choices=['cmake', 'ament_cmake'],
+            help='Which build tool to use')
+        parser.add_argument(
+            '--dependencies',
+            nargs='+',
+            default=[],
+            help='list of dependencies')
+        parser.add_argument(
+            '--maintainer-email',
+            nargs='?',
+            help='email address of the maintainer of this package'),
+        parser.add_argument(
+            '--maintainer-name',
+            nargs='?',
+            help='name of the maintainer of this package'),
+        parser.add_argument(
+            '--cpp-node-name',
+            nargs='?',
+            help='name of the empty cpp executable')
+        parser.add_argument(
+            '--cpp-library-name',
+            nargs='?',
+            help='name of the empty cpp library')
+
+    def main(self, *, args):
+
+        maintainer_name = getpass.getuser()
+        if not args.maintainer_name:
+            # try getting the name from the global git config
+            p = subprocess.Popen(
+                ['git', 'config', '--global', 'user.name'],
+                stdout=PIPE)
+            resp = p.communicate()
+            name = resp[0].decode().rstrip()
+            if name:
+                maintainer_name = name
+        else:
+            maintainer_name = args.maintainer_name
+
+        maintainer_email = maintainer_name + '@todo.todo'
+        if not args.maintainer_email:
+            # try getting the email from the global git config
+            p = subprocess.Popen(
+                ['git', 'config', '--global', 'user.email'],
+                stdout=PIPE)
+            resp = p.communicate()
+            email = resp[0].decode().rstrip()
+            if email:
+                maintainer_email = email
+        else:
+            maintainer_email = args.maintainer_email
+
+        if args.cpp_node_name == args.cpp_library_name:
+            raise ValueError('cpp_node_name has to be different from cpp_library_name')
+
+        print('going to create a new package')
+        print('package name:', args.package_name)
+        print('destination directory', args.destination_directory)
+        print('build tool:', args.build_tool)
+        print('maintainer_email:', maintainer_email)
+        print('maintainer_name:', maintainer_name)
+        print('dependencies:', args.dependencies)
+        print('cpp_node_name:', args.cpp_node_name)
+        print('cpp_library_name:', args.cpp_library_name)
```

Comment:
Some args might be `None`. Maybe it would be nicer to not print a line in that case?